src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -111,6 +118,10 @@ def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=No
                     for initFile in initFiles:
                         copyCommand += "source %s\n" % initFile
 
+        if (args.wma_disablewriterecovery or args.disablewriterecovery) and not\
+                (args.wma_cerncastor or args.cerncastor):
```

Comment:
Same comment as in the other PR, we can just remove that secondary check, cleaner that way.